optionals/tracers/stringtable.xml

```diff
@@ -0,0 +1,338 @@
+                <German>7.62mm 150 Schuss Kiste Leuchtspur (rot)</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_150Rnd_762x54_Box_Tracer_red_description">
+                <English>Caliber: 7.62x54 mm Tracer - Red&lt;br /&gt;Rounds: 150&lt;br /&gt;Used in: Zafir</English>
+                <German>Kaliber: 7.62x54 mm Leuchtspur - Rot&lt;br /&gt;Patronen: 150&lt;br /&gt;Verwendet in: Zafir</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_150Rnd_762x54_Box_Tracer_yellow">
+                <English>7.62mm 150Rnd Box Tracer (Yellow)</English>
+                <German>7.62mm 150 Schuss Kiste Leuchtspur (Rot)</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_150Rnd_762x54_Box_Tracer_yellow_description">
+                <English>Caliber: 7.62x54 mm Tracer - Yellow&lt;br /&gt;Rounds: 150&lt;br /&gt;Used in: Zafir</English>
+                <German>Kaliber: 7.62x54 mm Leuchtspur - Gelb&lt;br /&gt;Patronen: 150&lt;br /&gt;Verwendet in: Zafir</German>
+            </Key>
+
+            <Key ID="STR_ACE_Tracers_150Rnd_93x64_Mag_yellow">
+                <English>9.3mm 150Rnd Belt Mixed (Yellow)</English>
+                <German>9.3mm 150 Schuss Gurt gemischt (Gelb)</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_150Rnd_93x64_Mag_yellow_description">
+                <English>Caliber: 9.3x64mm&lt;br /&gt;Rounds: 150&lt;br /&gt;Used in: Navid</English>
+                <German>Kaliber: 9,3 x 64 mm&lt;br /&gt;Schuss: 150&lt;br /&gt;Verwendet für: Navid</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_150Rnd_93x64_Mag_red">
+                <English>9.3mm 150Rnd Belt Mixed (Red)</English>
+                <German>9.3mm 150 Schuss Gurt gemischt (Rot)</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_150Rnd_93x64_Mag_red_description">
+                <English>Caliber: 9.3x64mm&lt;br /&gt;Rounds: 150&lt;br /&gt;Used in: Navid</English>
+                <German>Kaliber: 9,3 x 64 mm&lt;br /&gt;Schuss: 150&lt;br /&gt;Verwendet für: Navid</German>
+            </Key>
+
+            <Key ID="STR_ACE_Tracers_130Rnd_338_Mag_yellow">
+                <English>.338 NM 130Rnd Belt Mixed (Yellow)</English>
+                <German>.338 NM 130 Schuss Gurt gemischt (gelb)</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_130Rnd_338_Mag_yellow_description">
+                <English>Caliber: .338 Norma Magnum&lt;br /&gt;Rounds: 130&lt;br /&gt;Used in: SPMG</English>
+                <German>Kaliber: .338 Norma Magnum&lt;br /&gt;Schuss: 130&lt;br /&gt;Verwendet für: SPMG</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_130Rnd_338_Mag_green">
+                <English>.338 NM 130Rnd Belt Mixed (Green)</English>
+                <German>.338 NM 130 Schuss Gurt gemischt (grün)</German>
+            </Key>
+            <Key ID="STR_ACE_Tracers_130Rnd_338_Mag_green_description">
```

Comment:
these are duplicates of vanilla string
```
        <Key ID="str_a3_cfgmagazines_130rnd_338_mag1">
          <Original>Caliber: .338 Norma Magnum&lt;br /&gt;Rounds: 130&lt;br /&gt;Used in: SPMG</Original>
```
I think we can just drop the description for all these and just use inheritance

Reply:
fixed in 8711f42